components/worktemplates/components/customize.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+
+import PublicPrivateSelector from 'components/widgets/public-private-selector/public-private-selector';
+import {Visibility} from '@mattermost/types/worktemplates';
+
+export interface CustomizeProps {
+    className?: string;
+    name: string;
+    visibility: Visibility;
+
+    onNameChanged: (name: string) => void;
+    onVisibilityChanged: (visibility: Visibility) => void;
+}
+
+const Customize = ({
+    name,
+    visibility,
+    onNameChanged,
+    onVisibilityChanged,
+    ...props
+}: CustomizeProps) => {
+    const {formatMessage} = useIntl();
+
+    return (
+        <div className={props.className}>
+            <div className='name-section-container'>
+                <p>
+                    <strong>
+                        {formatMessage({id: 'worktemplates.customize.name_title', defaultMessage: 'What project is this for?'})}
+                    </strong>
+                </p>
+                <p className='customize-name-text'>
+                    {formatMessage({id: 'worktemplates.customize.name_description', defaultMessage: 'This will help you and others find your project items. You can always edit this later.'})}
+                </p>
+                <input
```

Comment:
So that is what I did first, but:

- My HTML ended up being bigger (had to set `useLegend` to false)
- I ended up writing more CSS because it comes with a fieldset that contains border and background colors and padding

So in the end I felt like I was fighting the `<Input>` field more than I was using it 😅 .